src/plugins/search/plugin.search.js

```diff
@@ -61,22 +61,22 @@ BookReader.prototype.setup = (function (super_) {
     /** @type { {[pageIndex: number]: SearchInsideMatchBox[]} } */
```

Comment:
Note: Noticing the blue blobs not in the right place! Not sure if this is a regression; let's test in petabox review app, cause was broken in master for standalone br.

Reply:
perhaps we need to rederive this plato book as `goody`'s search result highlights align perfectly 🤔 
![image](https://user-images.githubusercontent.com/7840857/147992084-06a0b2b6-8eec-4ca7-9923-f2f08f54a686.png)

Reply:
will hold as none of this code has affected search result highlight positioning && `goody` returns properly aligned highlights